### ISSUE TITLE
partitions(aws-cn, us-gov) support

### ISSUE THREAD

**Issue by zxkane** [aws/aws-step-functions-data-science-sdk-python#120]
All resources of task are hard code partition `aws`. The partition info should be fetched from boto3 client.

**Comment by zxkane** [comment]
We are using below workaround for reference,

```python
from stepfunctions.steps.fields import Field

attrs=getattr(step, ‘fields’)
attrs[Field.Resource.value]=‘arn:aws-cn:states:::glue:startJobRun.sync’
setattr(step, ‘fields’, attrs)
```

**Comment by wong-a** [comment]
Noted. Thanks for raising this!

Setting `Field.Resource.value` for `Steps` such as [this one](https://github.com/aws/aws-step-functions-data-science-sdk-python/blob/master/src/stepfunctions/steps/service.py#L38) need to be updated.

**Comment by hlmiao** [comment]
How test the fixed version, same issue as previous experienced in cn-north-1.
{
  "error": "States.Runtime",
  "cause": "An error occurred while executing the state 'Extract, Transform, Load' (entered at the event id #2). The resource belongs to a different partition from the running execution."
}

**Comment by wong-a** [comment]
@hlmiao https://github.com/aws/aws-step-functions-data-science-sdk-python/pull/131 hasn't been merged or released yet. If you want to test it out, you can pull the source from the PR's branch.

**Comment by hlmiao** [comment]
Got it.

**Comment by wong-a** [comment]
Released in v2.1.0: https://github.com/aws/aws-step-functions-data-science-sdk-python/releases/tag/v2.1.0
